generators/client/templates/_bower.json

```diff
@@ -4,46 +4,46 @@
   "appPath": "<%= MAIN_SRC_DIR %>",
   "testPath": "<%= TEST_SRC_DIR %>spec",
   "dependencies": {
-    "angular": "1.5.5",
-    "angular-aria": "1.5.5",
-    "angular-bootstrap": "1.3.3",
+    "angular": "1.5.8",
+    "angular-aria": "1.5.8",
+    "angular-bootstrap": "2.0.0",
```

Comment:
this is a major version bump, we need to manually test the app to make sure stuff works as expected, coz our automated UI tests are not sufficient